### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by Awakhiwekyle**
Update on docs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,28 +4,40 @@ ChitChat is a Python-powered Social Network.
 
 ## Installation
 
-**TODO** Update the sections below:
+1. cd into backend folder
+    cd backend
```
Comment (terrameijar) [inline README.md:8]: Let's add markdown formatting for this:

i.e \`\`\`cd backend\`\`\`

File: README.md [hunk 2]
```diff
@@ -4,28 +4,40 @@ ChitChat is a Python-powered Social Network.
...
+
+2. create virtual environment
```
Comment (terrameijar) [inline README.md:10]: Please capitalise the "c" in create for uniformity.

File: README.md [hunk 3]
```diff
@@ -4,28 +4,40 @@ ChitChat is a Python-powered Social Network.
...
+    ### Installing virtualenv
+    ```pip install virtualenv```
```
Comment (terrameijar) [inline README.md:12]: Here you're installing the third party`virtualenv` but you don't use it, making it redundant. Let's remove this line.

Python now has a built-in `venv` module for managing virtual environments and that's exactly what you used below when you did ```python -m venv venv```. The -m flag tells Python to use a built in module.

Please remove the ```pip install virtualenv``` line.

File: README.md [hunk 4]
```diff
@@ -4,28 +4,40 @@ ChitChat is a Python-powered Social Network.
...
+
+    ### Creating virtual environment
+    ```python -m venv venv```
+
+    ### Activating the virtual environment
+    ```source venv/bin/activate```
```
Comment (terrameijar) [inline README.md:18]: This is good! Some of our users might be on Windows machines, so let's add instructions on how to activate the virtual environment on Windows. The command looks something like this:

```venv\Scripts\activate```

**Comment by terrameijar** [comment]
Thank you @Awakhiwekyle, your PR is good. We just need to tighten up a few things. I have left comments on the lines that need some work. Please make those changes, and then once done, do another push to this branch and we'll be good to go.

File: README.md (hunk 4 above)
Comment (Awakhiwekyle) [reply README.md:18]: Noted. Thanks Vuyie.

File: README.md (hunk 1 above)
Comment (Awakhiwekyle) [reply README.md:8]: Noted.

File: README.md (hunk 3 above)
Comment (Awakhiwekyle) [reply README.md:12]: Noted.

**Review by terrameijar — APPROVED** [review]
Perfect, thank you.

**Pull request merged**